mungegithub/mungers/flake-manager.go

```diff
@@ -127,6 +127,11 @@ func (p *FlakeManager) isIndividualFlake(f cache.Flake) bool {
 	return true
 }
 
+// makeGubernatorLink returns a URL to view the build results in a GCS path.
+func makeGubernatorLink(id string) string {
+	return "https://k8s-gubernator.appspot.com/build" + id
+}
```

Comment:
Is there some link about what this app is? What it does? How it is maintained? Breadcrumbs for the next poor sap?